src/api/app/views/webui2/webui/package/show.html.haml

```diff
@@ -2,6 +2,9 @@
   @pagetitle = "Show #{@project} / #{truncate(@package.name, length: 50)}"
   devel_package = @package.develpackage
 
+  content_for(:meta_title, @package.title.presence || @package.name)
```

Comment:
[Nitpick]: `@package.name` would also be written as `@package`, as it already happens with `@project.name`. The same applies to `src/api/app/views/webui/package/show.html.erb`.